db/migrate/20170824142643_drop_groups.rb

```diff
@@ -0,0 +1,12 @@
+class DropGroups < ActiveRecord::Migration[5.1]
+  def change
+    drop_table :groups do |t|
+      t.string :slug
```

Comment:
What does specifying the columns in the block for `drop_table` do?

Reply:
It allows a rollback of the migration